VX_integration.py

```diff
@@ -548,52 +580,65 @@ def DownloadVxData(self):
         
         self.UpdateProject(self.vxConnector.Project)
         
-    def connect_pushed(self):
+    def connect(self):
+        
         self.dlg.loading.setMovie(self.movie)
         self.movie.start()
-        if self.vxConnector.IsConnected and self.i == 0:
+            
+        if self.vxConnector.IsConnected:
                 if type(self.vxConnector.Project) != type(None): 
                     self.show_info(self.tr("Connected!")) 
                     if not self.layers_created:
                         self.create_layers(self.vxConnector.Project.CoordinateSystem)
                         
                     self.DownloadVxData()
-                    self.dlg.pushButton_2.setEnabled(True)
-                    self.dlg.reinitialize.setEnabled(True)
-                    self.i += 1
-        else:
-            self.show_error(self.tr("Connection failed! - Please try again"))
+            
+        self.dlg.pushButton_2.setEnabled(True)
+        self.dlg.reinitialize.setEnabled(True)
+        self.actions[2].setEnabled(True)
```

Comment:
please check text inside action, and not index in array

Reply:
Not able to - action do not have a specific name